backend/src/data/models/appointment.ts

```diff
@@ -0,0 +1,34 @@
+import { Sequelize, DataTypes, ModelCtor, Model } from 'sequelize';
+
+enum Type {
+  ONLINE = 'online',
+  OFFLINE = 'offline'
+}
+
+interface AppointmentInstance extends Model {
```

Comment:
We'll need this interface for all apps, so it would be better to move it to the `shared`. And then use the double extend
Something like `interface AppointmentInstance extends IAppointment, Model {}`. Do it for all models